Add details about Reports Admin update (#614)
silverstripe/developer-docs: en/08_Changelogs/5.4.0.md

- [Features and enhancements](#features-and-enhancements)
  - [Option to change `ClassName` column from enum to varchar](#classname-varchar)

>   - [Reports Quality of life updates](#reports-quality-of-life-updates)

Reviewer: ```suggestion
  - [Reports quality of life updates](#reports-quality-of-life-updates)
```

    Class: "DBClassNameVarchar('SilverStripe\\ORM\\DataObject', ['index' => false])"
```

### Reports quality of life updates

Numerous slight adjustments have been made to the [`ReportAdmin`](api:SilverStripe\Reports\ReportAdmin) class for a better experience.

The changes include:
- Search capability added (making use of [`PartialMatchFilter`](api:SilverStripe\ORM\Filters\PartialMatchFilter))
- Sorting by columns now posssible
- Default Sort is now `Title ASC`
- Reports list is now paginated

> - [`Description`](api:SilverStripe\Reports\Report::description()) is now displayed on in the list as a column

Reviewer: ```suggestion
- [`Description`](api:SilverStripe\Reports\Report::description()) is now displayed in the list as a column
```
Sorry, didn't notice this typo the first time around

    Class: "DBClassNameVarchar('SilverStripe\\ORM\\DataObject', ['index' => false])"
```

> ### Reports Quality of life updates

Reviewer: ```suggestion
### Reports quality of life updates
```

> Numerous slight adjustments have been made to the base ReportAdmin class for a better experience.

Reviewer: ```suggestion
Numerous slight adjustments have been made to the [`ReportAdmin`](api:SilverStripe\Reports\ReportAdmin) class for a better experience.
```
There are unlikely to be subclasses.
Also adding API link per https://docs.silverstripe.org/en/contributing/documentation/#links-to-api-documentation

> Such changes include:

Reviewer: ```suggestion
The changes include:
```

>  - Search capability added (Making use of `PartialMatchFilter`)
>  - Sorting by columns now posssible
>  - Pagination capability added
>  - `Description` is now displayed on in the list as a column
>  - Default Sort is now `Title ASC`

Reviewer: ```suggestion
 - Search capability added (making use of [`PartialMatchFilter`](api:SilverStripe\ORM\Filters\PartialMatchFilter))
 - Sorting by columns now posssible
 - Default Sort is now `Title ASC`
 - Reports list is now paginated
 - [`Description`](api:SilverStripe\Reports\Report::description()) is now displayed on in the list as a column
```
